src/components/blog/newsletter-signup.svelte

```diff
@@ -1,4 +1,5 @@
 <script lang="ts">
+  export let classNames = "";
```

Comment:
```suggestion
  let clazz = "";
  export { clazz as class };
```

☝️ allows us to use this component like regular HTML elements, i.e. `<NewsletterSignup class="mx-auto mt-large mb-huge" />`

Reply:
Thanks, @mikenikles for the great tip.